pkg/store/bucket.go

```diff
@@ -482,6 +532,26 @@ func (s *BucketStore) Info(context.Context, *storepb.InfoRequest) (*storepb.Info
 	}, nil
 }
 
+func (s *BucketStore) normalizeMinTime(mint int64) int64 {
```

Comment:
Can we rename it a bit? `normalize` does not tell much, I think we rather mean `limit` maybe?